win32ss/user/user32/windows/cursoricon.c

```diff
@@ -42,7 +42,9 @@ VOID LoadSystemCursors(VOID)
    }
 }
 
-/* This callback routine is called directly after switching to gui mode */
+/// This callback routine is called directly after switching to gui mode.
```

Comment:
Put this description just after the callback ID comment instead?

Reply:
Deleted comments.